Port note to IE11 users from EE docs (#11583)
metabase/metabase: docs/enterprise-guide/full-app-embedding.md

  For example, `https://*.metabase.com http://my-web-app.example.com:8080/`. Leaving this empty will default to a `frame-ancestors` value of `'none'`.
  If you're a fancy person, you can specify this URL in the environment variable `MB_EMBEDDING_APP_ORIGIN`.

> A note to IE11 users: Only the first URL will be valid for IE11 due to limitations in the HTTP headers it supports: IE11 does not support the `Content-Security-Policy` header, but supports `X-Frame-Options`. That header supports only a single value, though.

Reviewer: ```suggestion
A note to IE11 users: only the first URL will be valid for IE11 due to limitations in the HTTP headers it supports. IE11 does not support the `Content-Security-Policy` header, but does support `X-Frame-Options`, which can only accept a single value.
```
